handbook/openid-connect/openid-connect.md

```diff
@@ -0,0 +1,133 @@
+# Authenticating with OpenID Connect
+
+You may configure SystemLink to use [OpenID Connect](https://openid.net) for user authorization. This enables SystemLink to leverage corporate single sign-on (SSO) and the additional security benefits it provides such streamlined login and limiting the proliferation of user credentials. This also enables SystemLink to utilize a common identity for users across multiple applications. OpenID connect can be used in conjunction with or as a replacement for LDAP, Active Directory, and local Windows accounts for authentication. 
+
+
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2020R4 or greater. Please see [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server. 
+- A DNS name for the SystemLink server. 
+- SystemLink login with the **Server Administrator** role. 
+- Administrator desktop access to the SystemLink server
+- A OpenID Connect Provider server such as [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html), [Azure ADFS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs), [Okta](https://www.okta.com/openid-connect/), or another [certified provider](https://openid.net/certification/) that has been fully setup and configured for OpenID Connect authentication. 
+    - If you have not yet setup your provider please consult the vendor documentation for setup and configuration. 
+
+## Enabling OpenID Connect in SystemLink
+1. Log into the server running SystemLink and open **NI Web Server Configuration**
+2. Go to the **Authentication** tab and enable **Use OpenID Connect (advanced)**
+
+![Enable OpenID Connect in NI Web Server](odic-webserver.png)
+*Enable OpenID Connect in NI Web Server*
+
+## OpenID Configuration Files in SystemLink server
+
+There are three files that must be created to connect your SystemLink server to an OpenID Connect provider: `[provider-dns].conf`, `[provider-dns].client`, and `[provider-dns].provider`. The `[provider-dns]` portion of each filename must be the URL encoded fully qualified domain name. 
+
+> For example, a OpenID provider with the DNS `openid-provider.com` would yield files named `openid-provider.com/conf` , `openid-provider.com.client`, and `openid-provider.com.provider`. 
+
+These files do no exist for new SystemLink installations. Each file must be added to the `C:\Program Files\National Instruments\Shared\Web Server\conf\openidc` directory. The NI Web Server must be restarted for these changes to take effect.
```

Comment:
I believe the web server must be restarted for changes to any of the 3 files to take effect, not just this one.

Reply:
I clarified this.